src/App.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <h1>MEMORY GAMME</h1>
+  <button @click="openFullScreen" class="fullScreenButton"></button>
```

Comment:
skoro button otwiera i zamyka nie powinien nazywać się open tylko toggle
```suggestion
  <button @click="toggleFullScreen" class="fullScreenButton"></button>
```